src/AbstractNav.php

```diff
@@ -136,27 +118,26 @@ public function activateParents(bool $activate): static
         return $new;
     }
 
-    private function activateTree(NavItem|NavLink $item, NavLink ...$tree): void
+    private function activateTree(NavLink $link, NavLink ...$tree): void
     {
-        $isItem = $item instanceof NavItem;
-        $link = $isItem ? $item->getLink() : $item;
-        $tree[] = $link;
-
         if ($this->isLinkActive($link, null)) {
 
-            foreach ($tree as $navLink) {
-                $navLink->activate();
+            $link->activate();
```

Comment:
You can check here, it seems like a mistake.

Reply:
> You can check here, it seems like a mistake.

No, it`s not mistake. We activate current link if it not already (by index or url) and all it parents